pages/api/plans/index.test.js

```diff
@@ -25,7 +25,7 @@ describe('Create Plan Api', () => {
       firstName,
       lastName,
       systemIds,
-      initialUseAsPhp
+      hasPhp: initialUseAsPhp
```

Comment:
Should we maybe just renamed the const to `hasPhp` to avoid any ambiguity?

Reply:
Hm, it's also used on the line 38 as an expected response, I'll just change it to true